requirements.txt

```diff
@@ -1 +1,2 @@
 requests>=1.0
+retrying==1.3.3
```

Comment:
This is a library, it should set minimum supported version only. We let apps pin for specific versions.